source/trx2junit/Program.cs

```diff
@@ -39,22 +40,24 @@ private static async Task RunAsync(string[] args)
             Thread.CurrentThread.CurrentCulture   = CultureInfo.InvariantCulture;
             Thread.CurrentThread.CurrentUICulture = CultureInfo.InvariantCulture;
 
-            string trxFile   = args[0];
-            string jUnitFile = Path.ChangeExtension(trxFile, "xml");
-
-            Console.WriteLine($"Converting trx-file '{trxFile}' to JUnit-xml...");
+            Console.WriteLine($"Converting {args.Length } trx file(s) to JUnit-xml...");
             DateTime start = DateTime.Now;
 
-            var utf8 = new UTF8Encoding(false);
+            await Task.WhenAll(args.Select(Convert));
+
+            Console.WriteLine($"done in {(DateTime.Now - start).TotalSeconds} seconds. bye.");
+        }
 
+        private static async Task Convert(string trxFile){
+            string jUnitFile = Path.ChangeExtension(trxFile, "xml");
+            Console.WriteLine($"Converting '{trxFile}' to '{jUnitFile}'");
+            var utf8 = new UTF8Encoding(false);
```

Comment:
The encoding is for every file the same, so we could just reference a (static readonly) field instead of creating an instance for every file.

Reply:
updated